src/components/Settings.css

```diff
@@ -0,0 +1,11 @@
+.showBlock {
+  display:block;
+}
+
+.hideBlock {
+  display:none;
+}
+
+.settingsToggle {
+  cursor: pointer;
+}
```

Comment:
there a setting to add new line at the end of the file on save. you should add that lol